src/RobotHub.cpp

```diff
@@ -227,49 +227,63 @@ void RobotHub::onSimulationConfiguration(const proto::SimulationConfiguration &c
 }
 
 void RobotHub::handleRobotFeedbackFromSimulator(const simulation::RobotControlFeedback &feedback) {
-    proto::RobotData feedbackToBePublished;
-    feedbackToBePublished.set_isyellow(feedback.color == utils::TeamColor::YELLOW);
-
-    // proto::RobotFeedback* feedbackOfRobots = feedbackToBePublished.mutable_receivedfeedback();
-
-    for (auto const &[robotId, hasBall] : feedback.robotIdHasBall) {
-        proto::RobotFeedback *feedbackOfRobot = feedbackToBePublished.add_receivedfeedback();
-        feedbackOfRobot->set_id(robotId);
-        feedbackOfRobot->set_hasball(hasBall);
+    rtt::RobotsFeedback robotsFeedback;
+    robotsFeedback.source = rtt::RobotFeedbackSource::BASESTATION;
+    robotsFeedback.team = feedback.color == utils::TeamColor::YELLOW ? rtt::Team::YELLOW : rtt::Team::BLUE;
```

Comment:
What is the difference between utils::TeamColor and rtt::Team? Do we really need both?

Reply:
Previously, both AI and RobotHub had to define their own Team enum. I added a Teams enum in utils that can be used in all repos, but I did not remove the one in RobotHub yet. Ill do that now